index.d.ts

```diff
@@ -7,21 +7,21 @@
  * @see [source] {@link https://docs.snowflake.com/en/developer-guide/node-js/nodejs-driver}
  */
 declare module 'snowflake-sdk' {
-    export const enum RowMode {
+    export enum RowMode {
```

Comment:
Is it fix the problem? It works for isolatedModules? whether the fix was tested?

Reply:
I applied the union type instead of the enum.